docs/en/writing-running-appium/ios/ios-xctest-file-movement.md

```diff
@@ -13,14 +13,6 @@ Below is the basic format.
 
 ### Real device
```

Comment:
We probably should also remove the title as well

Reply:
No, since the following format is:

- real device
    - format (e.g. `documents`)
    - example
- simulators
    - format
    - example

Only we can remove ifuse installation section for now